github/ci/prow-deploy/molecule/default/collect.yml

```diff
@@ -34,8 +34,7 @@
 - name: Describe all in a single yaml.
   shell: |
     {{ kubectl_prow }} get all -o yaml > {{ artifacts_dir }}/get_all.yaml
-  environment:
-    KUBEVIRTCI_TAG: 2010091151-ea2c81d
```

Comment:
Where is this set now ?

Reply:
We use the `shell_environment` variable now, which includes `KUBEVIRTCI_TAG`, see the next line.

Reply:
OK, I didn't remember I put the TAG there too from the start.